src/extensions/cookie/bootstrap-table-cookie.js

```diff
@@ -212,12 +212,23 @@ const UtilsCookie = {
               element.value = cookie.text
               cachedFilters[cookie.field] = cookie.text
             } else if (cookie.text !== '' && element.tagName === 'SELECT') {
-              const option = document.createElement('option')
-              option.value = cookie.text
-              option.text = cookie.text
-              element.add(option, element[1])
-              element.selectedIndex = 1
-              cachedFilters[cookie.field] = cookie.text
+              var found = false
+              for (var i = 0; i < element.length; i++) {
+                var e = element[i]
```

Comment:
Please use `const` instead of `var`, also please rename the varible from `e` to e.g. `currentElement`

Reply:
sure